src/chrome/content/rules/University-of-Exeter.xml

```diff
@@ -26,6 +30,10 @@
 	<securecookie host="^(?:.*\.)?exeter\.ac\.uk$" name=".+" />
 
 
+	<!-- Cert only matches empslocal.ex.ac.uk -->
```

Comment:
I presume the comment is meaning it doesn't match empslocal.exeter.ac.uk (from testing), but I'm not sure what the reason for adding this is for.. The site forces HTTPS anyway it seems. Did you mean to rewrite from exeter to ex?